tron/serialize/runstate/DynamoDBStateStore.py

```diff
@@ -0,0 +1,132 @@
+import math
+import pickle
+from threading import Thread
+
+import boto3
+
+from tron.serialize.runstate.shelvestore import ShelveKey
+from tron.serialize.runstate.shelvestore import ShelveStateStore
+
+OBJECT_SIZE = 400000
+
+
+class DynamoDBStateStore(object):
+    def __init__(self, name, dynamodb_region) -> None:
+        self.dynamodb = boto3.resource('dynamodb', region_name=dynamodb_region)
+        self.client = boto3.client('dynamodb', region_name=dynamodb_region)
+        self.name = name
+        self.shelve = ShelveStateStore(name)
```

Comment:
For migration, we are writing to and reading from both DyanmoDB and BerkleyDB so that in case anything breaks, we could roll back without losing any data.

Reply:
don't think we should bother. this class is not that complex (especially if you remove all shelve-related stuff), we can test on stages and do a big-bang migration later.

Reply:
Since it's already implemented, I guess I can just use the current code and remove the shelve part once dynamodb works.